gensim/test/test_word2vec.py

```diff
@@ -1069,30 +1069,30 @@ def testLineSentenceWorksWithFilename(self):
     def testCythonLineSentenceWorksWithFilename(self):
         """Does CythonLineSentence work with a filename argument?"""
         from gensim.models import word2vec_corpusfile
-        with utils.smart_open(datapath('lee_background.cor')) as orig:
+        with utils.open(datapath('lee_background.cor'), 'rb') as orig:
             sentences = word2vec_corpusfile.CythonLineSentence(datapath('lee_background.cor'))
             for words in sentences:
                 self.assertEqual(words, orig.readline().split())
 
     def testLineSentenceWorksWithCompressedFile(self):
         """Does LineSentence work with a compressed file object argument?"""
-        with utils.smart_open(datapath('head500.noblanks.cor')) as orig:
+        with utils.open(datapath('head500.noblanks.cor'), 'rb') as orig:
             sentences = word2vec.LineSentence(bz2.BZ2File(datapath('head500.noblanks.cor.bz2')))
             for words in sentences:
                 self.assertEqual(words, utils.to_unicode(orig.readline()).split())
 
     def testLineSentenceWorksWithNormalFile(self):
         """Does LineSentence work with a file object argument, rather than filename?"""
-        with utils.smart_open(datapath('head500.noblanks.cor')) as orig:
-            with utils.smart_open(datapath('head500.noblanks.cor')) as fin:
+        with utils.open(datapath('head500.noblanks.cor'), 'rb') as orig:
+            with utils.open(datapath('head500.noblanks.cor'), 'rb') as fin:
                 sentences = word2vec.LineSentence(fin)
                 for words in sentences:
                     self.assertEqual(words, utils.to_unicode(orig.readline()).split())
 
     def testPathLineSentences(self):
         """Does PathLineSentences work with a path argument?"""
-        with utils.smart_open(os.path.join(datapath('PathLineSentences'), '1.txt')) as orig1,\
-        utils.smart_open(os.path.join(datapath('PathLineSentences'), '2.txt.bz2')) as orig2:
+        with utils.open(os.path.join(datapath('PathLineSentences'), '1.txt'), 'rb') as orig1,\
+        utils.open(os.path.join(datapath('PathLineSentences'), '2.txt.bz2'), 'rb') as orig2:
```

Comment:
I think in this case it makes more sense to nest:

```python
with utils.open(...) as orig1:
    with utils.open(...) as orig2:
        ...
```

instead of messing with line breaks.

Reply:
done